src/client.ts

```diff
@@ -151,7 +151,6 @@ export class SkynetClient {
   registry = {
     getEntry: getEntry.bind(this),
     getEntryUrl: getEntryUrl.bind(this),
-    getEntryLink: getEntryLink.bind(this),
```

Comment:
Why are we dropping `client.registry.getEntryLink()`?

Reply:
It can be a standalone function i.e. it doesn't require the client. I labeled that commit a breaking change.

Reply:
I came here to remark on the breaking change and then found this resolved conversation. Why was it resolved? I think ro-tex was proposing to keep `getEntryLink` as is and mark it as deprecated and add the async version alongside it?

Reply:
I got confused by the naming. It appeared as if we want from non async to async but it was async all along. The standalone one is not async 👍  approved in that case sorry